website/description_issues.qmd

```diff
@@ -25,3 +25,19 @@ Description: The goal of 'readr' is to provide a fast and friendly way to
     to flexibly parse many types of data found in the wild, while still
     cleanly failing when data unexpectedly changes.
 ```
+
+------------------------------------------------------------------------
+
+# Explaining Acronyms
+
+## Problem
+
+The automatic spell check has flagged an acronym as incorrect and results in a NOTE.
```

Comment:
Maybe extending this to:
"You are using acronyms in your DESCRIPTION file or they are flagged by the automatic spell check."
as sometimes acronyms are not flagged since they result in an actual word.